src/freenet/clients/fcp/RequestStatus.java

```diff
@@ -63,10 +69,15 @@ synchronized void restart(boolean started) {
 		this.totalBlocks = total;
 		this.minBlocks = min;
 		this.fetchedBlocks = fetched;
+        // clone() because Date is mutable
+		this.latestSuccess
```

Comment:
Whitespace fixed by 3418c7daa580672fa0239b41113ac27ec388f2d2.
Done this since I assume that people prefer having 1468 soon to having an answer about whether billing probably less than 5 EUR here is OK.